app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -4,11 +4,18 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  assignedContact: "",
+  tags: "",
 };
 
 export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
   title: yup.string().required("Title is required"),
   description: yup.string().required("Description is required"),
+  assignedContact: yup
+    .object()
+    .required("Assigned Contacts is required")
+    .nullable(),
```

Comment:
I am kinda confused here. What does it mean for an entity to be `required` as well as be `nullable`? Isn't it counter intuitive?

PS: If there's a valid business aspect for adding this logic, then we can keep it. I am just trying to understand the same.

Reply:
That's my bad. I was getting a null error and was trying out a few things. Turned out the issue was with the way I had declared the objects for these fields and I missed to remove the nullable here.